src/main/kotlin/team/karakum/component/Content.kt

```diff
@@ -58,3 +66,15 @@ val Content = FC<Props> {
         }
     }
 }
+
+private val visibleHeight: Length
```

Comment:
Think this calculations are redundant.

Now I've added Grid layout. 
Try to tune the grid to get `Content` component background to fill the entire grid cell